waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/model/ASTQueryMappingTest.java

```diff
@@ -37,13 +38,34 @@ public void transformOutboundDatabaseName() {
         + "FROM db1.table1 alias1 INNER JOIN db2.table2 alias2\n"
         + "ON alias1.field1 = alias2.field2";
 
-    assertEquals(
-        "SELECT * FROM "
+    assertThat(queryMapping.transformOutboundDatabaseName(metaStoreMapping, query),
+        is("SELECT *\n"
+            + "FROM "
             + PREFIX
-            + "db1.table1 alias1 JOIN "
+            + "db1.table1 alias1 INNER JOIN "
             + PREFIX
-            + "db2.table2 alias2  ON alias1.field1 = alias2.field2",
-        queryMapping.transformOutboundDatabaseName(metaStoreMapping, query));
+            + "db2.table2 alias2\n"
+            + "ON alias1.field1 = alias2.field2"));
+  }
+
+  @Test
+  public void transformOutboundDatabaseNameAliasWithBackTicks() {
+    ASTQueryMapping queryMapping = ASTQueryMapping.INSTANCE;
+    MetaStoreMapping metaStoreMapping = new MetaStoreMappingImpl(PREFIX, "mapping", null, null);
+
```

Comment:
Replace the HCom specific names with something more generic

Reply:
ah good point!